bin/start.sh

```diff
@@ -2,11 +2,19 @@
 
 set -e
 remote=0
-while getopts ":r" opt; do
+while getopts ":s:r" opt; do
```

Comment:
Will this work with multiple secrets?

Reply:
yeap, it reads thinks like `-s s1:value1 -s s2:value2`